README.md

```diff
@@ -0,0 +1,75 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Description the project.
+
+
+## Built With
+
+- Major languages
+- Frameworks
+- Technologies used
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
```

Comment:
- [ ] Kindly replace the information on the README template with the information on your project. 👇🏽 

> Please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.

- Clone this repository and open it in your browser could be a good instruction.